src/libcollections/vec_deque.rs

```diff
@@ -1073,7 +1073,7 @@ impl<T> VecDeque<T> {
         }
     }
 
-    /// Inserts an element first in the sequence.
+    /// Prepends an element to the front of the `VecDeque`.
```

Comment:
For `push_back` we have this sentence: Appends an element to the back of the `VecDeque`.

I just mirrored that wording for `push_front`. :) Do you have a better suggestion?